src/DynamoCore/DynamoCore.csproj

```diff
@@ -164,7 +164,7 @@ limitations under the License.
     <Compile Include="Core\AuthenticationManager.cs" />
     <Compile Include="Services\ShapewaysClient.cs" />
     <Compile Include="Models\NodeModelExtensions.cs" />
-    <Compile Include="Interfaces\ISource.cs" />
+    <Compile Include="Search\Interfaces\ISource.cs" />
```

Comment:
Let's get rid of the interfaces subfolder. I want namespaces that describe functionality, not type :)